torch/testing/_internal/common_utils.py

```diff
@@ -572,6 +577,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def noarchTest(fn):
```

Comment:
Comment for this decorator explaining what a "NoArch" test is and how the decorator is intended to be used? 

What's the goal of running these tests on only one architecture? If we want to run all meta tests on only a single architecture, for example, then we could just decline to add the Meta device to the list of available devices on CI builds other than the designated one.